Userland/Libraries/LibWeb/Painting/FilterPainting.cpp

```diff
@@ -135,4 +135,27 @@ void apply_backdrop_filter(PaintContext& context, Layout::Node const& node, Gfx:
     context.painter().blit(actual_region.location(), *backdrop_bitmap, backdrop_bitmap->rect());
 }
 
+void apply_filter(PaintContext& context, Layout::Node const& node, Gfx::FloatRect const& filter_rect, BorderRadiiData const& border_radii_data, CSS::FilterList const& filter)
+{
+    // FIXME: This is ad-hoc, as the filter effects spec doesn't seem to tell us how to apply the filter. This is basically the backdrop-filter steps, minus the backdrop root image and transform requirements.
```

Comment:
It looks like the filter effects spec references the SVG spec, which I think sort of matches my comment from before:
> SVG allows any painting operation to be filtered. (See [Filter Effects](https://www.w3.org/TR/SVG11/filters.html).)
>
> In this case the result must be as though the paint operations had been applied to an intermediate canvas initialized to transparent black, of a size determined by the rules given in [Filter Effects](https://www.w3.org/TR/SVG11/filters.html) then filtered by the processes defined in [Filter Effects](https://www.w3.org/TR/SVG11/filters.html).